src/conversations/Conversations.ts

```diff
@@ -429,6 +438,67 @@ export default class Conversations {
     return seenPeers
   }
 
+  async newGroupConversation(
+    context: InvitationContext
+  ): Promise<GroupConversation> {
+    const timestamp = new Date()
+
+    const initialMembers = context.metadata.initialMembers?.split(',')
+
+    if (!initialMembers) {
+      throw new Error('No initial members provided')
+    }
+
+    const members = await Promise.all(
+      initialMembers.map(async (member) => {
+        let contact = await this.client.getUserContact(member)
+        if (!contact) {
+          throw new Error(`Recipient ${member} is not on the XMTP network`)
+        }
+
+        // Coerce the contact into a V2 bundle
+        if (contact instanceof PublicKeyBundle) {
+          contact = SignedPublicKeyBundle.fromLegacyBundle(contact)
+        }
+
+        return toSignedPublicKeyBundle(contact)
+      })
+    )
+
+    const inviteResponses = await this.client.keystore.createInvites({
+      recipients: members,
+      context,
```

Comment:
I wonder if we want to enforce that the group ID has to go into the `conversationId`. It would make this compatible with the deterministic topic work. Otherwise, I can't add you to more than one group.